brightwind/analyse/analyse.py

```diff
@@ -326,17 +326,31 @@ def freq_table(var_series, direction_series, var_bin_array=np.arange(-0.5, 41, 1
 
 def time_continuity_gaps(data):
     """
-    Returns the gaps in timestamps for the data, that means that data isn't available for that period.
+    Returns the start and end timestamps of missing data periods. Also days lost.
+
+    A missing data period is one where data is not available for some consecutive timestamps. This breaks
+    time continuity of the data. The function returns a dataframe where the first column is the starting timestamp
```

Comment:
Explaining how the function works should explain that the sample period is automatically calculated using '_xxxxx' and this is then used to scroll? through the time series to find gaps. The reason for explaining all of this is the user will be able to figure out what might be going wrong if they send in data that has an inconsistent sampling period?